docroot/index.php

```diff
@@ -4,6 +4,8 @@
 define('CUR_PATH', dirname(__FILE__) . DIRECTORY_SEPARATOR);
 //define('TPL_PATH', CUR_PATH . 'templates' . DIRECTORY_SEPARATOR);
 
+$title_page = 'Home'

```

Comment:
Yeah, thanks for the PR in any case. We're now also synced with the upstream repo, so PRs can target the current state.